addons/website/static/src/snippets/s_tabs_images/000.scss

```
@@ -0,0 +1,10 @@
+.s_tabs_images {
+    .o_nav_tabs_description {
+        display: none;
+    }
+    .s_tabs_main.s_tabs_nav_with_decriptions.s_tabs_nav_vertical .nav-underline .o_nav_tabs_description {
```

This selector looks unnecessarily strong... Could you try something like this?
```suggestion
    .s_tabs_nav_with_decriptions:where(.s_tabs_nav_vertical) :where(.nav-underline) where(.o_nav_tabs_description) {
```